src/oop_ext/foundation/callback/_typed_callback.py

```diff
@@ -183,3 +183,70 @@ def Contains(
             func: Callable[[T1, T2, T3, T4, T5], None],
             extra_args: Sequence[object] = Callback._EXTRA_ARGS_CONSTANT,
         ) -> bool: ...
+
+
+class PriorityCallback0(Callback0):
+    if TYPE_CHECKING:
+        CallableType = Callable[[], None]
+
+        def Register(
+            self,
+            func: CallableType,
+            extra_args: Sequence[object] = Callback._EXTRA_ARGS_CONSTANT,
+            priority: int = 5,
+        ) -> "_UnregisterContext": ...
```

Comment:
I may be wrong, but shouldn't `PriorityCallback0` and others be a subclass of `PriorityCallback`?

Reply:
Good catch!

Reply:
Thanks a lot, this would have been embarrassing... :eyes